js/src/UXCapture.js

```diff
@@ -4,23 +4,30 @@ import View from "./View";
 const NOOP = () => {};
 
 export default class UXCapture {
-  constructor() {
-    this.onMark = NOOP;
-    this.onMeasure = NOOP;
+  onMark = NOOP;
+  onMeasure = NOOP;
 
+  /**
+   * Creates a UXCapture instance
+   * @constructor
+   */
+  constructor() {
     window.UX = this;
   }
 
   /**
-   * Sets expected marks and corresponding zones for current view
+   * Creates a new View instance and supplies expected marks and
+   * corresponding zones to the View.
+   *
+   * TODO: re-evaluate if we should allow multiple executions of this method
    *
-   * @TODO re-evaluate if we should allow multiple executions of this method
+   * @param {object} zoneConfigs
    */
   expect(zoneConfigs) {
-    // create a view object for initial, server-side rendered page view
+    // View object for initial, server-side rendered page view
```

Comment:
not a blocker, but this comment could also be updated to remove the reference to server-side rendering - there's nothing that inherently ties a View to server rendering